nltk/tree/prettyprinter.py

```diff
@@ -291,7 +291,7 @@ def dumpmatrix():
                 matrix[rowidx][i] = ids[m]
                 nodes[ids[m]] = tree[m]
                 # add column to the set of children for its parent
-                if m != ():
+                if len(m) > 0:
```

Comment:
```suggestion
                if len(m):
```

I believe `len(m)` would be equivalent to `len(m) > 0` in truthy context. Both only evaluate to False if `m` is an empty collection. (That said, I understand the argument that `len(m) > 0` could be simpler to understand)

Reply:
Totally agree, but like you said `len(m) > 0 `is simpler to understand, so I would personally go with it.

Reply:
Understandable!